.travis/deploy.sh

```diff
@@ -2,6 +2,18 @@ if [ ! -z "$TRAVIS_TAG" ]
 then
     echo "on a tag -> $TRAVIS_TAG and therefore we will do nothing. Tagged releases are deployed to sonatype manually."
```

Comment:
Change message to: "on a tag -> $TRAVIS_TAG and therefore we will do nothing. Tagged releases are deployed to sonatype and Docker Hub manually."